helm-chart/amalthea/values.yaml

```diff
@@ -88,7 +88,7 @@ image:
   repository: renku/amalthea
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: 0.1.0
```

Comment:
I think this should either unset or latest and we should make the ci action deploy "latest" on merges to master. But having 0.1.0 as the default will result in a really old version. I suspect the last "default" is that old but I dont know that this improves anything. So I would just not touch it. Same applies to all the other tags in the values file.